Neos.Error.Messages/Classes/Message.php

```diff
@@ -57,15 +57,15 @@ class Message
      * Constructs this error
      *
      * @param string $message An english error message which is used if no other error message can be resolved
-     * @param integer $code A unique error code
+     * @param integer|null $code A unique error code
      * @param array $arguments Array of arguments to be replaced in message
-     * @param string $title optional title for the message
+     * @param string|null $title optional title for the message
      * @api
      */
-    public function __construct(string $message, int $code = null, array $arguments = [], string $title = '')
+    public function __construct(string $message, int $code = null, array $arguments = [], $title = '')
```

Comment:
Adding typehint for string not possible without adding subtle changes in behaviour. Defaulting to `null` would change the value when nothing is passed and casting to string would change the value when `null` is explicitly passed (for whatever reason).